pandas/core/frame.py

```diff
@@ -6837,6 +6837,27 @@ def reorder_levels(self, order: Sequence[Axis], axis: Axis = 0) -> DataFrame:
         Returns
         -------
         DataFrame
+        
+        Examples
+        --------
+        >>> df = pd.DataFrame(
+        ...  {"name" : ['Rahul','Arijit']}, 
+        ...    index = [
+        ...        [90, 77], 
+        ...        ['CSE', 'EEE'],
+        ...        ['A','B']
+        ...    ]
+        ... )
+        >>> df
+                     name
+        90 CSE A    Rahul
+        77 EEE B   Arijit
+    
+        >>> df.reorder_levels([0,2,1])
```

Comment:
use spaces between then numbers (pep)

you can see if you can enable the doc-string test for this

Reply:
("Space between the numbers") Can you please explain this in more detail? as I am a beginner to the open-source contribution.

Reply:
I have updated the example. Please review it.

Reply:
you haven't put spaces between the numbers, please check https://www.python.org/dev/peps/pep-0008/